src/domain/IDList.dto.ts

```diff
@@ -0,0 +1,10 @@
+import { Field, InputType } from '@nestjs/graphql';
+
+@InputType()
+export class IdList {
+  @Field(() => [String], {
```

Comment:
Why a separate class rather than a straight array of strings like on the mutation replaceTagsOnTagset?

Reply:
Because I wasn't aware how to do it this way :) Thanks for pointing to that solution.